platform/windows/os_windows.cpp

```diff
@@ -1088,14 +1088,14 @@ void OS_Windows::initialize(const VideoMode& p_desired,int p_video_driver,int p_
 	RasterizerGLES3::make_current();
 #else
 	// FIXME: Does DX support still work now that rasterizer is no longer used?
- #ifdef DX9_ENABLED
-	rasterizer = memnew( RasterizerDX9(hWnd) );
- #endif
+	#ifdef DX9_ENABLED
```

Comment:
Actually it was correct previously, we keep `#ifdef` unintended, and the rest of the code within such conditionals should keep its natural indentation (one tab here).

When you fix it, could you squash the commits into the first one? :)